Index: src/by/tms/servlets/Autorization.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/by/tms/servlets/Autorization.java	(revision 8ace05392d4b61d6b9a95050fed988dfc7f4f528)
+++ src/by/tms/servlets/Autorization.java	(revision )
@@ -24,6 +24,7 @@
             ArrayList<Users> users = new ArrayList<Users>();
             if (login.trim().length() == 0 || login == null || password.trim().length() == 0 || password == null) {
                 req.getServletContext().getRequestDispatcher("/badAutorization.jsp").forward(req, resp);
+//                goToBadAutorization();
             } else {
                 for (Users user : Users.getUsers()) {
                     users.add(user);
@@ -43,9 +44,23 @@
                 req.getServletContext().getRequestDispatcher("/goodAutorization.jsp").forward(req, resp);
             } else {
                 req.getServletContext().getRequestDispatcher("/badAutorization.jsp").forward(req, resp);
+//                goToBadAutorization();
             }
         } catch (IOException e) {
             req.getServletContext().getRequestDispatcher("/badAutorization.jsp").forward(req, resp);
         }
     }
+
+
+//    public void goToBadAutorization () {
+//        ServletRequest req = null;
+//        ServletResponse resp = null;
+//        try {
+//             req.getServletContext().getRequestDispatcher("/badAutorization.jsp").forward(req, resp);
+//        } catch (ServletException e) {
+//            e.printStackTrace();
+//        } catch (IOException e) {
+//            e.printStackTrace();
+//        }
+//    }
 }
